test/utils.lua

```diff
@@ -0,0 +1,79 @@
+-- This module contains auxiliary functions for integration tests that use lua code.
+-- To use this module, you need to copy this file to the temp pytest directory for the duration
+-- of the integration tests, then use the lua require function in your instance cfg file.
+-- The copying of this file to pytest temdir is already implemented in class
+-- TarantoolTestInstance of test/utils.py module.
+-- Just use require('utils').bind_free_port(arg[0]) inside your cfg instance file.
+
+local box    = require('box')
+local ffi    = require('ffi')
+local socket = require('socket')
+
+local testutils = {}
+
+function testutils.get_bound_address_old_tarantool()
+    -- Get bound via box.cfg({listen = '127.0.0.1:0'}) socket addr for tarantool major version 1.
+    -- For tarantool major version higher then 1, this function can be replaced by box.info.listen.
+    local res = {}
+    for fd = 0, 65535 do
+        local addrinfo = socket.internal.name(fd)
+        -- Assume that the socket listens on 127.0.0.1.
+        local is_matched = addrinfo ~= nil and addrinfo.host == '127.0.0.1' and
+            addrinfo.family == 'AF_INET' and addrinfo.type == 'SOCK_STREAM' and
+            addrinfo.protocol == 'tcp' and type(addrinfo.port) == 'number'
+        if is_matched then
+            addrinfo.fd = fd
+            table.insert(res, addrinfo)
+        end
+    end
+    if #res ~= 1 then
+        error(('zero or more than one listening TCP sockets: %d'):format(#res))
+    end
+
+    -- We need only listening, not client sockets.
+    local value = ffi.new('int[1]')
+    local len   = ffi.new('size_t[1]', ffi.sizeof('int'))
+    local level = socket.internal.SOL_SOCKET
+    -- Filter socket with SO_REUSEADDR.
+    local reuseaddr = ffi.C.getsockopt(
```

Comment:
Why the `getsockopt()` success/error value is named `reuseaddr`? It is not a `reuseaddr` value and we already discussed it [here](https://github.com/tarantool/tt/pull/112#discussion_r975694086).

Reply:
Done, now value is named `status`